opentok/opentok-meet
# [OPENTOK-44633] Fix dependency opentok-editor

#### What is this PR doing?
Updating package-lock.json with a valid sub-dependency to avoid failures when npm i

#### How should this be manually tested?
In this PR you should be able to npm install with no errors. Check master branch npm install is throwing an error on opentok-editor dep.

#### What are the relevant tickets?
https://jira.vonage.com/browse/OPENTOK-44633
